### PR TITLE
Xdmod and release notes updates

### PR DESCRIPTION
**Pull request by ericfranz**
https://osc.github.io/ood-documentation-test/xdmod_and_release_notes_updates/

### CODE REVIEW

File: source/customization.rst [hunk 1]
```diff
@@ -1,4 +1,4 @@
-.. _customization:
+...................... _customization:
```
Comment (johrstrom) [inline source/customization.rst:1]: Is this some typo? It shows up as the literal `…………………. _customization:`

File: source/customization.rst [hunk 2]
```diff
@@ -966,3 +966,54 @@ Changes are applied by running ``update_ood_portal`` and restarting the ``ondema
...
+   .. code-block:: none
+
+      domains = "https://ondemand.osc.edu"
+
+#. Configure identity provider to include OnDemand host in HTTP Content-Security-Policy for frame-ancestors since OnDemand uses iFrames to trigger SSO with XDMoD when a user logs in
+
+   .. code-block:: none
+
+      frame-ancestors https://*.osc.edu 'self'
+
```
Comment (johrstrom) [inline source/customization.rst:1002]: Is this code example for KeyCloak or Dex?

Comment (ericfranz) [reply source/customization.rst:1002]: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Security-Policy/frame-ancestors

It is the value of the Content-Security-Policy header

It was what we had to set in Keycloak but I guess the full header would look like

    Content-Security-Policy: frame-ancestors https://*.osc.edu 'self'

File: source/customization.rst (hunk 1 above)
Comment (ericfranz) [reply source/customization.rst]: Fixed the typo. It no longer appears in the diff.

File: source/customization.rst (hunk 2 above)
Comment (ericfranz) [reply source/customization.rst:1002]: I added a link to MDN and clarified the example was the value of the Content-Security-Policy header that was used in Keycloak at OSC

**Pull request merged**